core/src/test/java/org/mskcc/oncotree/utils/CacheUtilTest.java

```diff
@@ -128,4 +133,105 @@ public void testRecacheWithBrokenOncoTreeVersionRepository() {
         config.resetNotWorkingVersionRepository(oncoTreeVersionRepository);
         cacheUtil.resetCache();
     }   
+
+    private void assertNoHistoryInChildren(Map<String, TumorType> tumorTypes) {
```

Comment:
these three functions are very similar. we could make a more general assert which takes the field to be tested as a parameter. (same comment for the assertXInChildren functions)